### ISSUE TITLE
Wayland wr-recorder additional recording file

### ISSUE THREAD

**Issue by botiapa** [ShayBox/Mon2Cam#43]
The wayland backend works, but currently creates a recording.mp4 in the working directory as well as outputs to /dev/videoX. I tried searching through the help page of it, but couldn't find a way to disable this.

Sorry for bothering you, but @brightcloudy, could lend a hand here please?

**Comment by ShayBox** [comment]
I don't see a way to disable it, so maybe a feature request upstream, but there's the -f flag, we can just point it to /dev/null to void it

**Comment by botiapa** [comment]
Yeah, but if I'm not mistaken, we already use the -f flag to point to to the virtual cam device:

``const commandLines = ["wf-recorder", "-x yuv420p", "-c rawvideo", "-m v4l2",`-f /dev/video${options.device}\`];``
